Gopkg.lock

```diff
@@ -44,10 +44,16 @@
   revision = "bcc4c8345a21301bf47c032ff42dd1aae2fe3027"
 
 [[projects]]
-  branch = "master"
-  name = "github.com/coreos/go-oidc"
-  packages = ["."]
-  revision = "a4973d9a4225417aecf5d450a9522f00c1f7130f"
+  name = "github.com/coreos/etcd"
```

Comment:
Is this some leftover? Or you added both postgres and etcd support?

Reply:
will remove